Sample/MediaStorageSample.xcodeproj/xcuserdata/tomoko.xcuserdatad/xcschemes/xcschememanagement.plist

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I think this file should not be committed because it's contained in `.gitignore` settings.